javaee-like-getting-started/src/test/java/io/quarkus/ts/javaee/gettingstarted/MapStructIT.java

```diff
@@ -0,0 +1,55 @@
+package io.quarkus.ts.javaee.gettingstarted;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.restassured.RestAssured;
+
+@QuarkusScenario
+public class MapStructIT {
+
```

Comment:
I would appreciate some context about the scenarios because this is first MapStruct addition here and having some context could help others with faster understanding.

I was thinking about JavaDoc details for test methods. Another option would be text in README, but I think having details close to the tests is the best.

Reply:
I will add few comments. I think it tests what it tests, there is no requirement on list of features and I just tried to cover as much as possible in not so much code.

Reply:
done